src/Workspaces/Core/Portable/Workspace/Solution/VersionStamp.cs

```diff
@@ -44,15 +44,16 @@ private VersionStamp(DateTime utcLastModified)
         }
 
         private VersionStamp(DateTime utcLastModified, int localIncrement)
+            : this(utcLastModified, localIncrement, GetNextGlobalVersion())
         {
-            _utcLastModified = utcLastModified;
-            _localIncrement = localIncrement;
-            _globalIncrement = GetNextGlobalVersion();
         }
 
         private VersionStamp(DateTime utcLastModified, int localIncrement, int globalIncrement)
```

Comment:
❓ Why is this not `DateTimeOffset`?

Reply:
that was not available when VersionStamp was created. if you want to change this to use DateTimeOffset, create new issue and send out PR.